examples/misc/lib/effective_dart/usage_good.dart

```diff
@@ -92,6 +92,26 @@ void miscDeclAnalyzedButNotTested() {
     // #enddocregion use-tear-off
   };
 
+  {
+    // #docregion default-separator
+    void insert(Object item, {int at = 0}) {
+      // ...
+    }
+    // #enddocregion default-separator
+  }
+
+  {
+    // #docregion default-value-null
+    void error([String message]) {
+      if (message != null) {
+        stderr.write(message);
+      } else {
+        stderr.write("\n");
+      }
+    }
```

Comment:
To match the suggestion above:

```dart
    void error([String message]) =>
        stderr.write(message ?? '\n');
```

Reply:
Done. (But kept braced body. Feels a little weird to explicitly override the formatter in a document that tells you to trust the formatter... 😄)

Reply:
Lol, no problem. Of course, we _are_ trusting the formatter. It's just that it can be tricky to balance code excerpt line length across many different and (mostly) unrelated code excerpts that _happen_ to be living in the same file. So we play with both the dartfmt line length argument and the occasional use of `//!<br>`. (All of this is FYI)